DynamoDB/CreateTable.js

```diff
@@ -0,0 +1,68 @@
+var AWS = require("aws-sdk");
+var keys = require("../keys.json");
+
+
+AWS.config.update({
+  region: keys.region,
+  endpoint: keys.endpoint
+});
+
+var dynamodb = new AWS.DynamoDB();
+
+var params = {
+    TableName : "V-Transfer",
+    KeySchema: [       
+        { AttributeName: "PK", KeyType: "HASH"},  //Partition key
+        { AttributeName: "SK", KeyType: "RANGE" }  //Sort key
+    ],
+    AttributeDefinitions: [       
+        { AttributeName: "PK", AttributeType: "S" },
+        { AttributeName: "SK", AttributeType: "S" },
+        { AttributeName: "u_id", AttributeType: "S" },
+        { AttributeName: "data", AttributeType: "S"}
+    ],
+    ProvisionedThroughput: {       
+        ReadCapacityUnits: 10, 
+        WriteCapacityUnits: 10
+    },
+    GlobalSecondaryIndexes: [{
+        IndexName: "Find_file",
+        KeySchema: [
+            {
+                AttributeName: "u_id",
```

Comment:
Here also

Reply:
In the global table we are using composite keys i.e. PK + SK, so is it possible that for GSI we only use PK, just confirm this.